website/docs/dbt-cli/installation.md

```diff
@@ -1,32 +1,133 @@
+
+## Homebrew
+
+Install [Homebrew](http://brew.sh/). Then, run:
+
+```shell
+brew update
+brew install git
+brew tap fishtown-analytics/dbt
+brew install dbt
+```
+
+Test your installation with `dbt --version`.
+
+### Upgrading
+
+To upgrade dbt, use:
+
+```shell
+brew update
+brew upgrade dbt
+```
+
+### Installing different versions with Homebrew
+
+You can install and use multiple versions of dbt with Homebrew through something called Homebrew "links." To allow installation of another version of dbt, first unlink the current version:
+
+```shell
+brew unlink dbt
+brew install dbt@0.17.0
+brew link dbt@0.17.0
+```
+
+Now, you can use dbt version 0.17.0:
+
+```shell
+$ dbt --version
+installed version: 0.17.0
+```
+
+You can switch between versions by linking the one you want to use:
+
+```shell
+brew unlink dbt@0.17.0
+brew link dbt
+```
+
+If you'd prefer to use the development version of dbt, you can install it as follows:
+
+```shell
+brew unlink dbt
+brew update
+brew install dbt-development
+brew link dbt-development
+```
 
 ## pip
 
-:::info Python3
+dbt is a Python module distributed on [pypi](https://pypi.org/project/dbt/), and can be installed via `pip`. We recommend using virtual environments when installing dbt via pip. Some additional steps are required for linux flavors. See the relevant section for [prerequisite commands](#additional-steps-by-installation).
 
+:::caution Python3
 The dbt CLI is compatible with Python versions 3.6 and higher. As of v0.15.0, dbt is no longer compatible with Python2.
-
 :::
 
-dbt is a Python module distributed on [pypi](https://pypi.org/project/dbt/), and can be installed via `pip`. We recommend using virtual environments when installing dbt via pip.
-## Installation
-```bash
+If you encounter SSL cryptography errors during installation, ensure your local `pip` is current (via [cryptography.io](https://cryptography.io/en/latest/faq/#compiling-cryptography-on-os-x-produces-a-fatal-error-openssl-aes-h-file-not-found-error))
+
+### Installation
+
+First, we recommend [python virtual environments](https://docs.python-guide.org/dev/virtualenvs/) to namespace `pip` modules. Here's an example setup:
+```shell
+python3 -m venv dbt-env				# create the environment
+source dbt-env/bin/activate			# activate the environment
+```
+
+If you install `dbt` in a virtual environment, that same virtual environment must be re-activated each time a shell window or session is created.
+
+Tip: `alias` the `source ...` command in your `$HOME/.bashrc`, `$HOME/.zshrc`, or whatever rc file your shell draws from. For example, you can add a command like `alias env_dbt='source <...>/bin/activate'`, where `<...>` is substituted for the path to your virtual environment configuration.
+
+```shell
 pip install dbt
 ```
 
-## Upgrading
+Check your installation with `dbt --version`.
+
+### Upgrading
 To upgrade dbt, use:
 ```
 pip install --upgrade dbt
-```
\ No newline at end of file
+```
+
+### Additional Steps by Installation
```

Comment:
We tend to use sentence case for titles. Also, should this say "operating system"?
```suggestion
### Additional steps by operating system
```